src/math.ts

```diff
@@ -0,0 +1,78 @@
+import express, {Request,Response,NextFunction} from "express";
+  const app = express();
+  
+  const validate = (req: Request, res: Response, next: NextFunction) => {
+    /*const {
+      params: { op, a, b },
+    } = req;
+    var x: number = +a;
+    var y: number = +b;
+  
+    if (
+      !Number.isNaN(x) &&
+      !Number.isNaN(y)
+    ) {
+      if (
+        op === "sum" ||
+        op === "sous" ||
+        op === "mult" ||
+        op === "div"
+      )
+        next();
+      else {
+        res.send(
+          "Error: you must choose sum or sous or mult ou div"
+        );
+      }
+    } else {
+      res.send(
+        "Error: a ou b not number"
+      );
+    }*/
+var num1 = Number(req.query.num1);
+var num2 = Number(req.query.num2);
+const op = req.params.op;
+
+if (op == "div" && num2 == 0) {
+  return res.status(400).json({ error: "division impossible sur zero" });
+}
+
```

Comment:
Consider thinking about what can happen if you got one of the numbers or the operator with an undefined value